src/error/display.rs

```diff
@@ -81,6 +84,7 @@ impl fmt::Display for EvalexprError {
             ModulationError { dividend, divisor } => {
                 write!(f, "Error modulating {} % {}", dividend, divisor)
             },
+            InvalidRegex { regex, message } => write!(f, "Regular expression {} is invalid: {}", regex, message),
```

Comment:
I would use {:?} for both arguments, since then the strings get surrounded by `"`, which makes the error message a bit easier to read in my opinion.
```suggestion
            InvalidRegex { regex, message } => write!(f, "Regular expression {:?} is invalid: {:?}", regex, message),
```

Reply:
Fixed.